File: fix_imports.py
import re

with open('src/pages/PublicDashboard.tsx', 'r') as f:
    content = f.read()

if 'recharts' not in content:
    content = "import { ResponsiveContainer, ScatterChart, CartesianGrid, XAxis, YAxis, Scatter, PieChart, Pie, Cell, Tooltip, Legend } from 'recharts';\n" + content

lucide_match = re.search(r"import \{([^}]+)\} from 'lucide-react';", content)
if lucide_match:
    imports = set([s.strip() for s in lucide_match.group(1).split(',')])
    imports.update(['Users', 'Clock', 'CheckCircle2'])
    new_lucide = f"import {{ {', '.join(imports)} }} from 'lucide-react';"
    content = content.replace(lucide_match.group(0), new_lucide)

with open('src/pages/PublicDashboard.tsx', 'w') as f:
    f.write(content)

File: fix_imports.js
const fs = require('fs');
let content = fs.readFileSync('src/pages/PublicDashboard.tsx', 'utf8');

if (!content.includes('from "recharts"')) {
    content = "import { ResponsiveContainer, ScatterChart, CartesianGrid, XAxis, YAxis, Scatter, PieChart, Pie, Cell, Tooltip, Legend } from 'recharts';\n" + content;
}

const lucideMatch = content.match(/import \{([^}]+)\} from 'lucide-react';/);
if (lucideMatch) {
    const imports = new Set(lucideMatch[1].split(',').map(s => s.trim()));
    ['Users', 'Clock', 'CheckCircle2'].forEach(i => imports.add(i));
    content = content.replace(lucideMatch[0], `import { ${Array.from(imports).join(', ')} } from 'lucide-react';`);
}

fs.writeFileSync('src/pages/PublicDashboard.tsx', content);
